tools/tcpaccept.py

```diff
@@ -88,16 +88,14 @@
 
```

Comment:
I haven't found a way to replace the above call to `bpf_probe_read` with a dereference, so the Jenkins' builds are going to fail. Pull request # "fixes" this by skipping all dereferences inside `bpf_probe_read` calls, regardless of their type (the last argument is not a `UnaryOperatorExpr` of type `AddrOf` here).